scheduler/core/service/service.py

```diff
@@ -239,6 +243,13 @@ def run(self,
         semesters = frozenset([Semester.find_semester_from_date(start_vis.datetime),
                                Semester.find_semester_from_date(end_vis.datetime)])
 
+        variant_per_site = None
+        if cc_per_site and iq_per_site:
+            variant_per_site = {site: Variant(cc=cc_per_site[site],
+                                              iq=iq_per_site[site],
+                                              wind_dir=Angle(0),
+                                              wind_spd=Quantity(0 * u.m / u.s)) for site in sites}
+
```

Comment:
This is now handled by the PR you approved yesterday... we get this information from the env service now.